pom.xml

```diff
@@ -1167,7 +1167,7 @@
 		<maven.compiler.plugin.version>3.8.0</maven.compiler.plugin.version>
 		<maven.liquibase.plugin.version>${liquibase.version}</maven.liquibase.plugin.version>
 		<maven.war.plugin.version>3.2.2</maven.war.plugin.version>
-		<maven.jetty.plugin.version>9.4.24.v20191120</maven.jetty.plugin.version>
```

Comment:
Had accidentally pushed the local changes

Reply:
Does this actually need to be updated though?

Reply:
Yup good idea to update it. Updated in b08491fd725ca9fd305283ad0e87df84ab7c1cd4